cognite/transformations_cli/__main__.py

```diff
@@ -1,7 +1,12 @@
+from dotenv import load_dotenv
+
 from cognite.transformations_cli.commands.base import transformations_cli
 
 
 def main() -> None:
+    # support local .env file with environment-variables
+    load_dotenv()
```

Comment:
I think this should work locally, not in CI/CD. 👍  We need to add a small test first before merging. I'll take a look soon.

Reply:
any update / roadmap how to support PRs going forward?

Reply:
sorry, i totally forgot 🤦 , let me try it locally and approve, can you also bump the versions in `pyproject.toml`, `cognite/transformations_cli/__init__.py` and `CHANGELOG.md`?

Reply:
@emelvarol-cognite merged with `main` and updated the version as you requested
- but there is a minor problem I see adding the `python-dotenv` dependency
- it **would be** included in latest branch of `cognite-extractor-utils` [2.2.0](https://github.com/cognitedata/python-extractor-utils)
- but transformation-cli has a quite outdated 

```
cognite-extractor-utils = "^1.6.2"
```

---
The latest ext-utils v2.2.0 would allow you as well to switch from (full packed)
```
cognite-sdk = "^2.53.1"
```
to lightweight sdk-core
```
cognite-sdk-core = {extras=["pandas"],version = "^2.56.0"}
```

do you think it is ok to update and test these dependencies with this PR too?

Reply:
Ofc sure! Thanks for contributing 🎉